apidoc/Titanium/Network/HTTPClient.yml

```
@@ -97,14 +97,52 @@ description: |
     In Titanium, if a connection with TLS 1.2 fails, Titanium will re-attempt the connection with
     TLS 1.1 and TLS 1.0. By default, TLS 1.2 is attempted first.
 
-    Use the <Titanium.Network.HTTPClient.tlsVersion> property to set the version of the TLS protocol
+    In android, use the <Titanium.Network.HTTPClient.tlsVersion> property to set the version of the TLS protocol
     if you know the version the server is running.  If you do not know, do not set this property.
-    Titanium will not fallback with a lower TLS version if the `tlsVersion` property is set.
+
+    In iOS, add `NSExceptionMinimumTLSVersion` key in plist section of tiapp.xml to set minimum TLS version. 
+    See following example-
```

```suggestion
    See following example:
```

The examples also feels kind of redundant. I think only the last one should be enough. Also make sure to use proper indentation and fenced code blocks.

```
    ```
    <key>NSAppTransportSecurity</key>
    <dict>
      <key>NSExceptionDomains</key>
      <dict>
        <key>www.appcelerator.com</key>
        <dict>
          <key>NSExceptionMinimumTLSVersion</key>
          <string>TLSv1.3</string>
        </dict>
      </dict>
    </dict>
    ```
```